src/Http/Http.Extensions/gen/StaticRouteHandlerModel/EndpointParameter.cs

```diff
@@ -65,6 +100,8 @@ public EndpointParameter(IParameterSymbol parameter, WellKnownTypes wellKnownTyp
     public string? AssigningCode { get; }
     public string HandlerArgument { get; }
     public bool IsOptional { get; }
+    public bool IsParsable { get; }
+    public Func<string, string, string> ParsingBlockEmitter { get; }
```

Comment:
Nit: Let's take advantage of the `MemberNotNullWhen` attribute to communicate that `ParsingBlockEmitter` will always be set if `IsParsable` is true.